core/package.json

```diff
@@ -52,7 +53,7 @@
     "next": "15.0.4-canary.18",
     "next-auth": "5.0.0-beta.25",
     "next-intl": "^3.26.0",
-    "nuqs": "^2.2.2",
+    "nuqs": "^1.20.0",
```

Comment:
Need to update `nuqs` to latest version in VIBES

Reply:
VIBES: https://github.com/makeswift/vibes/pull/413
Catalyst: handled in this PR's latest commit